### PR TITLE
Family Page HIFI

### PR DESCRIPTION
**Pull request by amlimjoc**

### CODE REVIEW

File: src/pages/ProgramsPage/FamilyPage/FamilyNavbar.jsx
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/FamilyNavbar.jsx:1]: theres a navbar that exists in components that you can utilize, i know during the time of developing this it wasn't nice looking but just for future reference use that one!

File: src/pages/ProgramsPage/FamilyPage/FamilyNavbar.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+function Navbar({ bg, title, subtitle, homeNav = false }) {
+  const navigate = useNavigate();
+
+  const handleNavigation = (path) => {
+    navigate(path);
+  };
+
+  return (
+    <>
+      <div className="relative">
```
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/FamilyNavbar.jsx:32]: make this cover the whole screen so you would add somethign like h-screen

File: src/pages/ProgramsPage/FamilyPage/WhatIsFam.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { useEffect, useRef } from "react";
+import { useIsVisible } from "../../../components/useIsVisible"; // Adjust the path as necessary
+
+const WhatIsFusion = () => {
+    const ref = useRef();
+    const isVisible = useIsVisible(ref);
+
+    return (
+        <div className="bg-white w-full h-[487.5px] flex items-center justify-center">
+            <div ref={ref} className={`flex flex-col items-center justify-center w-[816px] h-[258px] transition-opacity ease-in duration-700 ${isVisible ? "opacity-100" : "opacity-0"}`}>
```
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/WhatIsFam.jsx]: Add some more padding for the top and the bottom

File: src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      </div>
+      <div className="absolute right-0 top-0 ">
+        <img src={RightSquare} alt="Right Square"/>
+      </div>
+
+      {/* Content */}
+      <div className="relative z-10 flex flex-col items-center">
+        <div className='pt-24' ref={ref}>
+            <div className="flex items-center justify-center mb-2">
+                <div className={`border-t border-[#FAC80A] w-[115px] border-[2px] mr-2 transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}></div>
```
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx]: for these lines make them a bit longer like in figma

File: src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+import LeftSquare from '../../../images/LeftSquare.png';
+import RightSquare from '../../../images/RightSquare.png';
+import { useIsVisible } from "../../../components/useIsVisible"; // Adjust the path as necessary
+
+// Reusable FamSection component
+const FamSection = ({ title, image, isVisible }) => {
+  return (
+    <div className={`flex flex-col items-center transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}>
+      <img className="w-[300px] h-[200px] object-cover rounded-lg" src={image} alt={`${title} Image`} />
+      <h2 className="mt-4 text-2xl font-bold text-white">{title}</h2>
```
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx]: Make this less bold more thin like in the figma

File: src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+      {/* Content */}
+      <div className="relative z-10 flex flex-col items-center">
+        <div className='pt-24' ref={ref}>
+            <div className="flex items-center justify-center mb-2">
+                <div className={`border-t border-[#FAC80A] w-[115px] border-[2px] mr-2 transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}></div>
+                <h1 className={`text-5xl font-karla font-[700] leading-[52px] text-white transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}>‘23 - ‘24 Fams</h1>
+                <div className={`border-t border-[#FAC80A] w-[115px] border-[2px] ml-2 transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}></div>
+            </div>
+            <p className={`font-inter mt-4 text-center w-[50rem] font-[400] text-[16px] leading-8 transition-opacity duration-700 ease-in-out ${isVisible ? 'opacity-100' : 'opacity-0'}`}>
```
Comment (benson-fm) [inline src/pages/ProgramsPage/FamilyPage/OrangeFam.jsx]: make the width of this text a little longer so we can get that effect on the figma with the first line being longer than the other

**Review by benson-fm — CHANGES_REQUESTED** [review]
Great job Arvin! Just minor changes to do!

**Comment by amlimjoc** [comment]
closes #36

**Pull request closed without merging**